### ISSUE TITLE
Error: declaration-colon-newline-after  ]

### ISSUE THREAD

**Issue by Undistraction** [styled-components/vscode-styled-components#93]
This is OK:

```
const Layout = styled.div`
  ${example({ alpha: 1, bravo: { charlie: 3 } })};
`
```

This is flagged as an error:
```
const Layout = styled.div`
  ${example({ 
    alpha: 1, 
    bravo: { 
      charlie: 3,
    },
  })};
`
```

Error:

> [stylelint] Expected newline after ":" with a multi-line declaration (declaration-colon-newline-after)
> [stylelint] Expected no more than 0 empty lines (value-list-max-empty-lines)
> [stylelint] Expected no more than 0 empty lines (value-list-max-empty-lines)
> [stylelint] Expected no more than 0 empty lines (value-list-max-empty-lines)
> [stylelint] Expected no more than 0 empty lines (value-list-max-empty-lines)

![cap](https://user-images.githubusercontent.com/161489/40734621-df73fc7c-6430-11e8-9672-da11573950ae.jpg)

**Comment by Undistraction** [comment]
This is a problem with `stylelint-processor-styled-components`
